ProcessLib/ComponentTransport/ComponentTransportProcess.h

```diff
@@ -132,11 +138,16 @@ class ComponentTransportProcess final : public Process
         const double dxdot_dx, const double dx_dx, GlobalMatrix& M,
         GlobalMatrix& K, GlobalVector& b, GlobalMatrix& Jac) override;
 
+    void setCoupledSolutionsOfPreviousTimeStep();
+
     ComponentTransportProcessData _process_data;
 
     std::vector<std::unique_ptr<ComponentTransportLocalAssemblerInterface>>
         _local_assemblers;
 
+    /// Solutions of the previous time step
+    std::array<std::unique_ptr<GlobalVector>, 2> _xs_previous_timestep;
```

Comment:
What is the meaning of `_xs` in the variable name?

Reply:
Simply follow the implementation in other processes... _xs indicates that we have multiple solutions.